### ISSUE TITLE
ToastNotification component: in high contrast (HCM) colors of all types are the same

### ISSUE THREAD

**Issue by amatecka** [carbon-design-system/carbon#1123]
## Detailed description

> Describe in detail the issue you're having.

http://carbondesignsystem.com/components/notification/code
When I switch to the high contrast mode in Windows, the color of ToastNotification is always the same, for example white, regardless of the notification type. So the user is unable to see whether it is an alert or an informational message.
In the normal contrast mode, there are 3 colors: red/yellow/green.

> Is this issue related to a specific component?

Yes, ToastNotification (without an icon)

> What did you expect to happen? What happened instead? What would you like to see changed?

Colors should be kept: red/yellow/green.

> What browser are you working in?

Firefox 62 for example.

> What version of the Carbon Design System are you using?

carbon-components-9.1.1
carbon-components-react-5.52.3

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

IBM BigFix Inventory.

## Steps to reproduce the issue

Switch to the high contrast mode and open http://carbondesignsystem.com/components/notification/code

## Additional information

- Screenshots or code

![image](https://user-images.githubusercontent.com/43415169/45764104-39dd6f00-bc32-11e8-9395-49f7959cd169.png)

**Comment by dakahn** [comment]
This is tagged v9, but could be a bigger issue. @joshblack I know we are currently addressing high contrast mode upstream in our elements package. I assume that would help out these toast notifications as well? 💪🏽

**Comment by joshblack** [comment]
@dakahn it definitely could, right now it's only for icons but we could do it for other stuff as well if there is a good strategy here 👍

**Comment by dakahn** [comment]
Gave this a retest on Windows 10 latest build HCM default settings
![2019-06-13 16_19_36-Carbon Design System - Firefox Developer Edition](https://user-images.githubusercontent.com/40970507/59468302-4b395d80-8df7-11e9-9127-fc0798d804c3.png)

Looks like our iconography is taking the reset HCM CSS and accurately displaying. 👍
